.github/workflows/MERGE_QUEUE.yml

```
@@ -36,20 +36,60 @@ jobs:
+              macOS) DESTINATION="platform=macOS,arch=arm64" ;;
+              catalyst) DESTINATION="platform=macOS,arch=arm64,variant=Mac Catalyst" ;;
+            esac
+          fi
+          
+          echo "Using destination: $DESTINATION"
+          
           xcodebuild \
             -scheme Afluent-Package \
-            -destination '${{ matrix.destination }}' \
+            -destination "$DESTINATION" \
             clean build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)